public/model/map/layer_operations.ts

```diff
@@ -48,16 +48,19 @@ export const updateLayerVisibility = (map: Maplibre, layerId: string, visibility
   });
 };
 
-export interface LineLayerSpecification {
+export interface Layer {
```

Comment:
Shall we name it mbLayer or add some comment here to explain it's the layer in maplibre, because there are two layers conception: maps dashboards layer and maplibre layer

Reply:
Made it local and added comment. This layer is an abstraction between Dashboard and Maplibre.